### PR TITLE
Feature/operators

### PR DESCRIPTION
**Pull request by sabalkaw**
Add systemverilog operators under operators section in systemverilog.ts file
Add testcases for operators under test.ts

### CODE REVIEW

File: src/systemverilog/systemverilog.test.ts [hunk 1]
```diff
@@ -110,5 +110,43 @@ testTokenization('systemverilog', [
...
+	[{
+		line: '!n',
+		tokens: [
+			{ startIndex: 0, type: 'delimiter.sv'},
+			{ startIndex: 1, type: 'identifier.sv'}
+		]
+	}],
+
+	[{
+		line: 'assign r = c + i;',
```
Comment (xadegunt) [inline src/systemverilog/systemverilog.test.ts:135]: I think Spencers LRM examples has a lot of good examples that we're going to want to leverage for operators. (operators in particular has a lot of weird ways they can be used so we'll want to make sure to test out those edge cases)

here are some examples to get you started:

    initial begin
        #3 cb.v <= expr1; // Matures in cycle 1; equivalent to ##1 cb.v <= expr1

            assert (req1 || req2)

    arrayB[i][j+:2] == arrayA[k][m-:2]

            ##1 (!b[*0:$] ##1 b && (data_out == lv), v1 = lv)

    assert property (@(clk) go ##1 get[*2] |-> !stop throughout put[->2]);

        mul.i = (a.r * b.i) + (a.i * b.r);

        2'b0z: result = flaga ? 'x : 0;

                    (reset_n && ($past(en) != 0) && en == 0)
                    |->(en == 0)[*min_quiet:max_quiet]
